### PR TITLE
feat: Update README and CONTRIBUTING

### PR DESCRIPTION
**Pull request by segfaultxavi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,43 @@
...
+```
+
+## Contributing
+
+Before contributing please [read the CONTRIBUTING instructions](CONTRIBUTING.md).
+
+## Getting Help
+
+- [NIS Developer Documentation](https://nemproject.github.io/nem-docs).
+- [NIS Technical Reference](htt.ps://nemproject.github.io/nem-docs/pages/Whitepapers/NEM_techRef.pdf)
```
Comment (Jaguar0625) [inline README.md:37]: https

File: README.md [hunk 2]
```diff
@@ -1,11 +1,43 @@
...
+The main folders are:
+
+| Folder                       | Content                                            |
+| ---------------------------- | -------------------------------------------------- |
+| `org.nem.core.connect`       | HTTP connection handling.                          |
+| `org.nem.core.crypto`        | Hashing, Key management, signing and verification. |
+| `org.nem.core.math`          | Mathematical helpers.                              |
+| `org.nem.core.model`         | Building blocks like Transactions or Blocks.       |
+| `org.nem.core.serialization` | Serialization helpers.                             |
+| `org.nem.core.time`          | Time and time synchronization methods.             |
```
Comment (Jaguar0625) [inline README.md:18]: capitalization seems inconsistent 

how did you determine main folders?

File: README.md [hunk 3]
```diff
@@ -1,11 +1,43 @@
...
+## Building the package
+
+The package uses [Apache Maven](https://maven.apache.org/) and the ``pom.xml`` file is initially configured to work with **Java 8**. To work on more recent versions of Java run the ``travis_prepare.sh`` script first.
+
+Then build and install the package as usual:
+
+```bash
+mvn install
+```
+
```
Comment (Jaguar0625) [inline README.md:29]: probably would add information about running `mvn test` and `mvn spotless` here

File: README.md (hunk 2 above)
Comment (segfaultxavi) [reply README.md:18]: "Transaction" and "Block" have a special meaning so I consider them proper nouns and hence the capitalization. But it's no big deal. Removed.

@0x6861746366574 asked for _some_ package structure, _however small_. Unless someone gives me this summary I am afraid I do not have the time to read ALL source folders and find out their purpose. I just chose the ones that seemed more obvious to me, as a starting point. Feel free to provide the rest :)

File: README.md (hunk 3 above)
Comment (segfaultxavi) [reply README.md:29]: :+1:

**Comment by Jaguar0625** [comment]
async - utilities for working with async timers
function - generic functional prototypes
i18n - handling of Unicode (UTF8) files
math - linear algebra matrix and vector utilities
messages - implementation of transfer plain and encrypted messages
metadata - handling of application metadata and facility for embedding it into the jar
model - basic data types for transactions and states
node - basic data types for node information
serialization - binary and json (de)serialization implementations and serialization utilities  
time - basic data types for time synchronization
utils - other utilities (collections, encoders, etc)

**Review by Jaguar0625 — APPROVED** [review]

**Pull request merged**